engines/mtropolis/runtime.h

```diff
@@ -2044,6 +2044,7 @@ class WorldManagerInterface : public RuntimeObject {
 	int32 _opInt;
 	bool _gameMode;
 	bool _combineRedraws;
+	bool _postponeRedraws;
```

Comment:
Please initialize this to false in the constructor.

Reply:
Done.